redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -105,6 +122,9 @@
         return $list->getColumnLink('deleteType', '<i class="rex-icon rex-icon-delete"></i> ' . rex_i18n::msg('media_manager_type_delete'));
     });
 
+    $list->addColumn('copyType', '<i class="rex-icon rex-icon-copy"></i> ' . rex_i18n::msg('media_manager_type_copy'), -1, ['', '<td class="rex-table-action">###VALUE###</td>']);
```

Comment:
ich würde diese aktion nicht als "letzte aktion" in der liste aufführen. für mich gehört "löschen" immer ganz rechts hin.

ich würde "kopieren" zwischen "editieren" und "löschen" sehen